apps/meteor/ee/app/license/server/startup.ts

```diff
@@ -26,3 +27,19 @@ License.setLicenseLimitCounter('privateApps', () => getAppCount('private'));
 License.setLicenseLimitCounter('marketplaceApps', () => getAppCount('marketplace'));
 // #TODO: Get real value
 License.setLicenseLimitCounter('monthlyActiveContacts', async () => 0);
+
+const updateCronJob = async () => {
+	if (License.hasValidLicense() === (await cronJobs.has('licenseLimitChecker'))) {
+		return;
+	}
+
+	if (License.hasValidLicense()) {
+		await cronJobs.add('licenseLimitChecker', '*/30 * * * *', () => License.revalidateLimits());
+	} else {
+		await cronJobs.remove('licenseLimitChecker');
+	}
+};
+
+License.onValidateLicense(async () => updateCronJob());
+License.onInvalidateLicense(async () => updateCronJob());
+void updateCronJob();
```

Comment:
on startup, won't this make this function called 2 times? Maybe we can trust in one of the license hooks, or are you thinking in other scenario?